bin/oneoff/add_progress_v2_closed_beta_users

```diff
@@ -24,9 +24,13 @@ CSV.table(teacher_id_csv, encoding: 'iso-8859-1:utf-8').by_col[:id].each do |id|
     next
   end
 
-  if user.teacher?
+  next unless user.teacher?
+  begin
     user.update!(progress_table_v2_closed_beta: true)
     count += 1
+  rescue ActiveRecord::RecordInvalid
+    puts "RecordInvalid error #{id}"
+    next
```

Comment:
`next` is not strictly necessary here because nothing else happens before the end of the loop, but it's fine as is.

```suggestion
```